dashboard/config/blocks/GamelabJr/gamelab_makeNumSprites.json

```diff
@@ -0,0 +1,24 @@
+{
+  "category": "Sprites",
+  "config": {
+    "func": "makeNumSprites",
+    "inline": false,
+    "blockText": "make {NUM} new {ANIMATION_NAME} sprites",
+    "color": [
+      355,
+      ".7",
+      ".7"
+    ],
+    "args": [
+      {
+        "name": "NUM",
+        "type": "Number",
+        "field": true
+      },
+      {
+        "name": "ANIMATION_NAME",
+        "customInput": "costumePicker"
+      }
+    ]
+  }
+}
```

Comment:
Does the linter need a new line at the end of this file?

Reply:
No, all of the block config files are auto-generated through the levelbuilder UI and don't have the extra line at the end.

Reply:
TIL! Thanks!